### PR TITLE
Day/04/phase1

### PR DESCRIPTION
**Pull request by ppeigne**

### CODE REVIEW

File: day03/ex09/ex09.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+ ### Exercise 09 - Regularized Logistic Regression
```
Comment (afoures) [inline day03/ex09/ex09.md]: Should be `#` not `###`

File: day03/ex10/ex10.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Exercise 04 - Z-score standardization
```
Comment (afoures) [inline day03/ex10/ex10.md]: -> Exercise 10

File: day03/ex10/ex10.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex04              |
```
Comment (afoures) [inline day03/ex10/ex10.md]: same

File: day03/ex09/ex09.md (hunk 1 above)
Comment (Talasta) [reply day03/ex09/ex09.md]: I will fix this after.

**Comment by ppeigne** [comment]
🍾

**Pull request merged**